Moreinfo.php

```diff
@@ -0,0 +1,24 @@
+<html>
+    <head>
+        <title>about</title>
+        <link rel="stylesheet" href="styles.css?Time=<?php echo microtime()?>" />
+    </head>
+    <body>
+        <div class="headerbluebackground">
```

Comment:
We'll talk more about this later, but when you need to write multiple words without spaces in between, most people capitalize the first letter of each word after the first. So this would be written `headerBlueBackground`. This is called "camel casing" and it just makes the code a bit easier to read.